lib/shoulda/matchers/active_record/uniqueness.rb

```diff
@@ -1,14 +1,14 @@
 module Shoulda
   module Matchers
-    module ActiveRecord
+    module ActiveModel
       # @private
```

Comment:
Later I intend to create a PR to send this change and rename the file name which is different from the constant if that's ok with you.